pkg/scheduler/framework/plugins/interpodaffinity/plugin.go

```diff
@@ -52,6 +55,74 @@ func (pl *InterPodAffinity) Name() string {
 	return Name
 }
 
+func (pl *InterPodAffinity) Requeue(ctx context.Context, p *v1.Pod, event framework.ClusterEvent, oldObj, obj interface{}) framework.SchedulingHint {
+	if !event.In(pl.EventsToRegister()) {
+		// not interested event.
+		return framework.PodNotAffected
+	}
+
+	switch event.Resource {
+	case framework.Pod:
+		return pl.requeueByPodEvent(ctx, p, event, oldObj, obj)
+	case framework.Node:
+		// TODO(sanposhiho): implement it
```

Comment:
don't forget to open an issue if we decide to go with this approach

Reply:
OK. Once we've merged it, I'll open the PR to support `Requeue` in in-tree PreEnqueue + Filter plugins.

Reply:
~~the PR~~ -> the issue